test/gtest/ucp/ucp_test.cc

```diff
@@ -320,9 +319,10 @@ ucp_test_base::entity::~entity() {
 }
 
 void ucp_test_base::entity::connect(const entity* other,
-                                    const ucp_ep_params_t* ep_params_cmn) {
+                                    const ucp_ep_params_t* ep_params_cmn,
+                                    bool reuse_worker_idx) {
```

Comment:
This is backward compatibility for multi-threading tests, they do connections N:N, i.e. N eps to N workers with the same indexes. Otherwise, other tests also may require changes.